h/migrations/versions/1ef80156ee4_add_sidebar_tutorial_dismissed_column_.py

```diff
@@ -0,0 +1,28 @@
+"""Add the sidebar_tutorial_dismissed column to user table.
+
+Revision ID: 1ef80156ee4
+Revises: 43645baa68b2
+Create Date: 2015-12-21 18:49:15.688177
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = '1ef80156ee4'
+down_revision = '43645baa68b2'
+
+from alembic import op
+import sqlalchemy as sa
+
+
+def upgrade():
+    with op.batch_alter_table('user') as batch_op:
+        batch_op.add_column(sa.Column('sidebar_tutorial_dismissed',
+                                      sa.Boolean,
+                                      nullable=False,
+                                      default=False,
+                                      server_default=sa.sql.expression.true()))
```

Comment:
The `server_default` here is `true()` so that on a db migration, the tutorial will not be shown to all existing users. But the app default is `False` so it will be shown to new users who register thereafter.